bin/augur

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import uuid
+import pprint
+import click
+import augur
+
+
+@click.command()
+@click.argument('config', type=str)
+@click.option('-v', '--verbose', is_flag=True)
+def run(config, verbose):
+    """Here comes a Roman guy with a spade in his hand.
+    """
+    print("Watch out! Here comes a Roman guy with a ",
+          "spade in his hand!", flush=True)
+
+    # do we actually need this here?
+    # Might be useful to pass this to fc
+    analysis_id = uuid.uuid4().hex
+    print('analysis id:', analysis_id, flush=True)
```

Comment:
Firecrown will produce its own ID to identify each run in some cases. I don't think with the way you are calling it, it will do this since none of the data hits disk.